src/main/java/seedu/address/model/person/TimeTable.java

```diff
@@ -0,0 +1,28 @@
+package seedu.address.model.person;
+
+/**
+ * Represents a timetable that is associated with a person
+ *
+ */
+public class TimeTable {
+
+    //Is this even correct syntax, i cries java is hard
+    private TimeSlot[][] weeklyslots = new TimeSlot[7][24];
```

Comment:
Probably the new goes inside the constructor

Also something something constructive comments lol